src/main/java/uk/ac/ebi/eva/pipeline/ParameterFromProperties.java

```diff
@@ -0,0 +1,68 @@
+package uk.ac.ebi.eva.pipeline;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.ac.ebi.eva.pipeline.parameters.JobParametersNames;
+
+import java.lang.reflect.Field;
+import java.util.Properties;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Service
+public class ParameterFromProperties {
+
+    private static final Logger logger = LoggerFactory.getLogger(ParameterFromProperties.class);
+    private static final String PROPERTY = "${";
+    private static final String OR_NULL = ":#{null}}";
+    private static final String PROPERTY_ID_REGEX = "(?<=\\$\\{).*(?=:#\\{null})";
+
+    @Value(PROPERTY + JobParametersNames.DB_NAME + OR_NULL)
+    private String databaseName;
+
+    @Value(PROPERTY + JobParametersNames.DB_COLLECTIONS_VARIANTS_NAME + OR_NULL)
+    private String collectionVariantsName;
+
+    @Value(PROPERTY + JobParametersNames.INPUT_STUDY_ID + OR_NULL)
```

Comment:
If no value found, then null to don't create a jobParameter with null value in the job scope.